pkg/validation/internal/manifests.go

```diff
@@ -0,0 +1,70 @@
+package internal
+
+import (
+	"fmt"
+
+	"github.com/operator-framework/api/pkg/validation/errors"
+
+	"github.com/operator-framework/operator-registry/pkg/registry"
+)
+
+type ManifestsValidator struct{}
```

Comment:
I suppose this is a fairer name given this validator's current logic. However I'd rather keep the name general in case the constituent manifest types of a manifests directory are changed, or other, non-update graph validation logic is added in the future. How about `VersionedManifestSetValidator` or `ManifestsDirValidator`?

Alternatively we could add future logic as described above in another validator and reserve this one for validating the update graph.

Reply:
> Alternatively we could add future logic as described above in another validator and reserve this one for validating the update graph.

This is more where my head is at, since I view validators as composable units that form larger validators (if `Validators` implemented `Validator`). I can also see where it can be preferable to make larger units to avoid duplication of work or handle interdependencies. If that's the case, then either of the names you've suggested are fine with me.

Reply:
Ehh because this code is only validating the update graph right now, we can name this `PackageUpdateGraphValidator` and add another validator later. These names are subject to change since they're internal types.